codejail_service/codejail.py

```diff
@@ -11,11 +11,11 @@ def safe_exec(code, input_globals, **kwargs):
 
     input_globals is not mutated, unlike in the codejail library.
 
-    Returns a tuple of (globals_dict, emsg).
+    Returns a tuple of (globals dict, error message).
 
-    - globals_dict is the globals dictionary that resulted from execution,
+    - globals dict: The globals dictionary that resulted from execution,
       whether or not an error was raised
-    - emsg is an error message string, or None if execution succeeded
+    - error message: An error message string, or None if execution succeeded
```

Comment:
Do we want the underscores?

Reply:
I decided it was better not to have them, since they're not actually named variables, just slots in a return tuple.

Reply:
Ok. Makes for more consistent code, but agreed it isn't strictly necessary. Thanks.